src/popup.js

```diff
@@ -30,13 +30,13 @@ window.onload = function() {
 
       chrome.storage.sync.set({ profiles: data, rawtext: rawstr },
         function() {
-          msgSpan.innerHTML = '<span style="color:#1111dd">Updated configuraton!</span>';
+          msgSpan.innerHTML = '<span style="color:#1111dd">Configuration has been updated!</span>';
           setTimeout(function() {
             msgSpan.innerHTML = '';
           }, 2500);
         });
     } catch (e) {
-      msgSpan.innerHTML = '<span style="color:#dd1111">Failed to save because of invalid format!</span>';
+      msgSpan.innerHTML = '<span style="color:#dd1111">Failed to save configuration because format was invalid!</span>';
```

Comment:
@jbonachera Is the original sentence something wrong? I want the explanation to be short because it is placed in a limited area.